storage/ndb/src/kernel/blocks/dbtup/DbtupAbort.cpp

```diff
@@ -545,6 +545,50 @@ int Dbtup::TUPKEY_abort(KeyReqStruct * req_struct, int error_type)
     jam();
     terrorCode = ZUNSUPPORTED_BRANCH;
     break;
+  /*
```

Comment:
I rewrote this function to simply directly use terrorCode = error number, but can fix that when I merge with your changes.

Reply:
ok